File: ase/misc/clr/form2.lua

local Form = CLR.Load("System.Windows.Forms",
  "System.Windows.Forms.Form");
local Application = CLR.Load("System.Windows.Forms",
  "System.Windows.Forms.Application");
local x = Form();
x.Text = "ASE-CLR bridge";
x.Show();
Application.Run(x)

File: ase/misc/clr/form2.pl

my $Form = $CLR->Load("System.Windows.Forms",
  "System.Windows.Forms.Form");
my $Application = $CLR->Load("System.Windows.Forms",
  "System.Windows.Forms.Application");
my $x = $Form->NewInstance();
$x->{Text} = "ASE-CLR bridge";
$x->Show();
$Application->Run($x);

